### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by dweindl**
Bring code quality back to A.

### CODE REVIEW

File: tests/test_parameter_mapping.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
+import numpy as np
 import pandas as pd
 import petab
```
Comment (yannikschaelte) [inline tests/test_parameter_mapping.py]: alphabetic ordering + own tools / petab last? very in favor.

File: petab/lint.py [hunk 2]
```diff
@@ -239,7 +241,7 @@ def assert_parameter_estimate_is_boolean(parameter_df):
     0 or 1.
     """
     for estimate in parameter_df['estimate']:
-        if not int(estimate) in [True, False]:
+        if int(estimate) not in [True, False]:
```
Comment (yannikschaelte) [inline petab/lint.py]: duh

File: petab/core.py [hunk 3]
```diff
@@ -971,25 +972,25 @@ def flatten_timepoint_specific_output_overrides(
         unique_noise = df["noiseParameters"].unique()
 
         # Loop
-        for i in range(len(unique_noise)):
-            for j in range(len(unique_sc)):
-                # Find the position of all instances of unique_noise[i]
+        for i_noise, cur_noise in enumerate(unique_noise):
```
Comment (yannikschaelte) [inline petab/core.py]: yessssssss

File: tests/test_parameter_mapping.py (hunk 1 above)
Comment (dweindl) [reply tests/test_parameter_mapping.py]: yeah
pycharm does the job for you (code -> optimize imports)

**Pull request merged**